modules/mono/editor/bindings_generator.cpp

```diff
@@ -3676,6 +3685,7 @@ void BindingsGenerator::_populate_global_constants() {
 
 			if (enum_name != StringName()) {
 				EnumInterface ienum(enum_name);
+				// TODO: ienum.is_flags is always false for core constants since they don't seem to support bitfield enums
```

Comment:
Maybe you could add the relevant code anyway, this should be implemented for core constants soon.

Reply:
I'm not sure how this would be exposed in CoreConstants, should I add something like this?

```diff
struct _CoreConstant {
#ifdef DEBUG_METHODS_ENABLED
	StringName enum_name;
+	bool is_bitfield = false;
	bool ignore_value_in_docs = false;
#endif
	const char *name = nullptr;
	int64_t value = 0;
}

class CoreConstants {
public:
+	static bool get_global_constant_is_bitfield(int p_idx);
};
```